datahub-web-react/src/app/entity/shared/tabs/Documentation/components/DescriptionEditor.tsx

```diff
@@ -140,12 +143,9 @@ export const DescriptionEditor = ({ onComplete }: { onComplete?: () => void }) =
                     <CheckOutlined /> Save
                 </Button>
             </TabToolbar>
-            <StyledMDEditor
-                value={updatedDescription}
-                onChange={(v) => handleEditorChange(v || '')}
-                preview="live"
-                visiableDragbar={false}
-            />
+            <EditorContainer>
+                <Editor content={updatedDescription} onChange={(v) => handleEditorChange(v)} />
```

Comment:
Yup we could! But not sure if the editor would render well in a modal. Perhaps in the next iteration?

Reply:
would you mind if we try and see how it looks in the modal? hopefully we can make some adjustments if it's not looking great, but I just think it would be confusing to have two different rich text editors in the app. Especially when one is much nicer than the other - people may be disappointed when trying to edit schema field documentation from the modal for example.

Reply:
Hey @chriscollins3456, I've pushed a new commit that reflects the new editor across the app, looks good so far. Let me know what you think!

Reply:
amazing! thanks so much :)